src/mono/wasm/templates/templates/browser/index.html

```diff
@@ -9,6 +9,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="modulepreload" href="./main.js" />
   <link rel="modulepreload" href="./dotnet.js" />
+  <link rel="preload" href="./mono-config.json" as="fetch" crossorigin="anonymous">
```

Comment:
I wouldn't put it in to the template. I'm not even sure whether to have modules here.
I understand why it is good to have it, but I don't see it as something that must be in every "sample" / app.

Reply:
I think these 4 preloads are good compromise
@radekdoulik please comment as you requested to add it here.

Reply:
I think these 2 make sense. It was visible in the profiler, that it helped with the loading times. It will be interesting to see how it will affect the times we measure for app start.

Reply:
I do understand it, but do we want these lines in every app that is created?

Reply:
Yes :)